theforeman.org/pipelines/test/deb_packaging.groovy

```diff
@@ -0,0 +1,61 @@
+def packages_to_build
+def build_steps
+
+pipeline {
+    agent { label 'debian' }
+
+    options {
+        timestamps()
+        timeout(time: 4, unit: 'HOURS')
+        ansiColor('xterm')
+        buildDiscarder(logRotator(daysToKeepStr: '7'))
+    }
+
+    stages {
+        stage('Clone Packaging') {
+            steps {
+
+                deleteDir()
+                ghprb_git_checkout()
+
+            }
+        }
+
+        stage('Find Packages to Build') {
+            steps {
+
+                script {
+                    def version = ghprbTargetBranch == 'deb/develop' ? 'nightly' : ghprbTargetBranch.replace('deb/', '')
+                    packages_to_build = find_changed_debs("origin/${ghprbTargetBranch}")
+
+                    build_steps = build_deb_package_steps(packages_to_build, version, ghprbPullAuthorLogin, binding.hasVariable('ghprbPullId'))
```

Comment:
is there a case where the *test* pipeline is triggered without a PR? you do rely on `ghprbPullAuthorLogin`, so IMHO we can just assume that `pull_request=true`.

Reply:
Yes, that is true now. When I wrote this I was still potentially re-using this for the release pipeline.